### PR TITLE
Albert - US10 : Marriage After 14

### PR DESCRIPTION
**Pull request by achen3680**

### CODE REVIEW

File: project03.py [hunk 1]
```diff
@@ -232,7 +232,14 @@ def verifyDeathBefore150YearsOld(person):
         age = death_unix - bday_unix
     years_in_seconds = 150 * 365 * 24 * 60 * 60
     return age <= years_in_seconds
-
+  
+def marriageAfter14(person1, person2):
+    #check if both people are above 14 years of age to marry
+    if (person1['age'] > 14 and person2['age'] > 14):
```
Comment (DQSevilla) [inline project03.py]: This doesn't solve the user story, it must verify that **marriage** happens between of-age individuals. You are also not actually verifying the input gedcom file in main

**Review by DQSevilla — CHANGES_REQUESTED** [review]

File: project03.py [hunk 2]
```diff
@@ -233,12 +233,18 @@ def verifyDeathBefore150YearsOld(person):
...
     return age <= years_in_seconds
   
-def marriageAfter14(person1, person2):
-    #check if both people are above 14 years of age to marry
-    if (person1['age'] > 14 and person2['age'] > 14):
-        return True
-    else:
-        return False
+def verifyMarriageAfter14(family):
+    if not family['divorced'] == 'NA':
```
Comment (DQSevilla) [inline project03.py]: No need for a divorce check

File: project03.py [hunk 3]
```diff
@@ -233,12 +233,18 @@ def verifyDeathBefore150YearsOld(person):
...
+        wife = individualsDict[family['wifeId']]
+        husband = individualsDict[family['husbandId']]
+        
+        #if both wife and husband are over 14 years of age
+        if (wife['age'] > 14 and husband['age'] > 14):
```
Comment (DQSevilla) [inline project03.py]: Not correct, must be 14 years after birth, not currently 14 years of age

**Review by DQSevilla — CHANGES_REQUESTED** [review]

File: project03.py (hunk 3 above)
Comment (DQSevilla) [reply project03.py]: To be clear, print out a helpful error message every time:
- A families marriage date minus the wife's birthday was less than 14 years (unix time)
- A familieis marriage date minus the husbamd's birthday was less than 14 years (unix time)

**Review by Jarbo52 — APPROVED** [review]
lgtm

**Review by DQSevilla — APPROVED** [review]
Fix conflicts and merge

**Pull request merged**